kickstarts/partials/packages/includes.ks.erb

```diff
@@ -10,7 +10,9 @@ ipmitool
 # ruby dependencies auto-installed by ruby-install
 # prefer to have these dependencies explicitly required in the kickstart
 # https://github.com/postmodern/ruby-install/blob/8957e0d1a52d59b45cf48cef8f15a1d8ed921cd4/share/ruby-install/ruby/dependencies.txt
```

Comment:
Yes these are ruby's dependencies, but even so, I wonder if the ruby-install rpm should preload those dependencies for us, because you can't build ruby without them (and the fact that it's an rpm implies this is a fedora/rhel platform target).